MediaBrowser.Common/Net/NetworkExtensions.cs

```diff
@@ -136,6 +137,61 @@ public static string FormatIpString(IPAddress? address)
             return str;
         }
 
+        /// <summary>
+        /// Try parsing an array of strings into subnets, respecting exclusions.
+        /// </summary>
+        /// <param name="values">Input string to be parsed.</param>
+        /// <param name="result">Collection of <see cref="IPNetwork"/>.</param>
+        /// <param name="negated">Boolean signaling if negated or not negated values should be parsed.</param>
+        /// <returns><c>True</c> if parsing was successful.</returns>
+        public static bool TryParseSubnets(string[] values, out Collection<IPNetwork> result, bool negated = false)
+        {
+            result = new Collection<IPNetwork>();
```

Comment:
We shouldn't be using Collection tbh

Reply:
What's a better alternative? I don't really know much about the performance/allocations of those collection types.